README.rst

```diff
@@ -6,10 +6,10 @@ The PyMC developers have forked Theano to a new project called Aesara that is be
 
 
 To install the package, see this page:
-   http://deeplearning.net/software/theano/install.html
+   https://theano-pymc.readthedocs.io/en/latest/install_windows.html
```

Comment:
Good idea. 
Here you point to the windows install file. Not the general install file.